django/conf/__init__.py

```diff
@@ -145,7 +146,7 @@ def _show_deprecation_warning(self, message, category):
         # LazyObject __getattribute__(), -4 the caller.
         filename, _, _, _ = stack[-4]
         if not filename.startswith(os.path.dirname(django.__file__)):
-            warnings.warn(message, category, stacklevel=2)
+            warnings.warn(message, category, skip_file_prefixes=django_file_prefixes())
```

Comment:
I wondered what this function was for and realized it's no longer used, so I made a removal PR: https://github.com/django/django/pull/19901